### PR TITLE
[MAR-724] universnal nav bar redesign

### PR DESCRIPTION
**Pull request by darkul1**
### Please review these reminders and either check the box or explain why not:

- [x] Read and followed the [contributing guidelines](https://eds.eks.dev.ethos-int.com/#/Guidelines?id=section-contribute)?
- [ ] Component is exported from [index.js](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.js)?
- [ ] Type is exported from [index.d.ts](https://github.com/getethos/ethos-design-system/blob/master/src/components/index.d.ts) so Typescript users can consume it? Added a test and ran `yarn test:types`?
- [x] Bumped the yarn version?

### Referencing PR or Branch (e.g. in CMS or monorepo):
# [Deploy Preview](https://deploy-preview-7366--ethos-cms.netlify.app/)

### Screenshots and extra notes:
image

image

### CODE REVIEW

**Review by arnas007 — APPROVED** [review]

File: src/components/UniversalNavbarExpanded/CtaButton.module.scss [hunk 1]
```diff
@@ -1,4 +1,5 @@
 @import '../Media/Media.scss';
+@import '../Variables.scss';
```
Comment (coleEthos) [inline src/components/UniversalNavbarExpanded/CtaButton.module.scss:2]: Why did this need to be added?

**Review by coleEthos — COMMENTED** [review]
Overall code LGTM. Will wait until final design QA from Loc has been addressed and CMS PR has been updated with latest (both consumer and agents menu links) before approving. Nice work Darius!

**Review by coleEthos — CHANGES_REQUESTED** [review]
Can we eliminate the shifting of the text when bolded, so it functions like the old bolding of text when hovered? After that I think this is good to go!

See in attached screen recording how the text shifts left to right when bolded upon hover, but does not move on staging.

https://github.com/getethos/ethos-design-system/assets/87672141/0a009820-6236-4fd3-8c46-00f651662906

**Comment by darkul1** [comment]
> Can we eliminate the shifting of the text when bolded, so it functions like the old bolding of text when hovered? After that I think this is good to go!
> 
> See in attached screen recording how the text shifts left to right when bolded upon hover, but does not move on staging.
> 
>  Screen.Recording.2023-06-02.at.5.40.46.PM.mov

@coleEthos resolved this!

**Review by coleEthos — CHANGES_REQUESTED** [review]
Could we make one more minor change here? Make the clickable area the button only, so you don't accidentally click this CTA when expanding/closing accordion menus on the right side of the screen? I did this once when testing on my phone. 

Screen Shot 2023-06-08 at 10 57 14 PM

**Review by coleEthos — APPROVED** [review]
LGTM, remember to add git tag with version number as described in readme, before or after merge!

**Pull request merged**